### DISCUSSION TITLE
coc-toml extension

### DISCUSSION THREAD

**Discussion by kkiyama117** [neoclide/coc.nvim#2735] [Show and tell]
# About
I made extension for toml language, which is based on [taplo](https://github.com/tamasfe/taplo), library written in Rust.
This is new package and there are few probrems, derived from both of `coc-toml` and `taplo`, but almost all this plugin works well. I want to get feedback to implove it and fix bugs.

## url
- [npm](https://www.npmjs.com/package/coc-toml)
- [repo](https://github.com/kkiyama117/coc-toml)

**Comment by kkiyama117** [comment]
@chemzqm @fannheyward @oblitum 
I request you to add `coc-toml` to wiki because almost all works are done. But I don't know anyone can change wiki. I'll take care of it if I can, but I was wondering if you could do it if not.

**Reply by fannheyward** [reply]
The wiki https://github.com/neoclide/coc.nvim/wiki/Using-coc-extensions is open to edit.

**Reply by kkiyama117** [reply]
Thank you for your apply. I did it.
